### PR TITLE
rawx: quorum write mode

### PR DESCRIPTION
**Pull request by jkasarherou**

### CODE REVIEW

File: src/main/java/io/openio/sds/storage/rawx/RawxClient.java [hunk 1]
```diff
@@ -218,149 +204,149 @@ public void deleteChunks(List<ChunkInfo> l) {
...
+				}
+				throw e;
+			}
+		}
+		consume(data, size, gens, futures);
+
+		try {
+			ArrayList<UploadResult> successes = new ArrayList<UploadResult>();
+			for (Future<UploadResult> f : futures) {
+				UploadResult result = f.get();
```
Comment (fvennetier) [inline src/main/java/io/openio/sds/storage/rawx/RawxClient.java:301]: Why did you get rid of the timeout?

Comment (jkasarherou) [reply src/main/java/io/openio/sds/storage/rawx/RawxClient.java:301]: I don't see why such a timeout should be used to limit the time of a single chunk complete upload.

Comment (fvennetier) [reply src/main/java/io/openio/sds/storage/rawx/RawxClient.java:301]: This was a safeguard against stuck upload tasks. We can't configure write timeout on Java sockets.

Comment (jkasarherou) [reply src/main/java/io/openio/sds/storage/rawx/RawxClient.java:301]: I understand but this safeguard is more harmful, it should not apply to the whole upload operation.

**Pull request merged**